mozilla-services/consvc-shepherd
# [SHEP-80] Sync deals in the Renewal and Verbal stages to the database

## References

[SHEP-80](https://mozilla-hub.atlassian.net/browse/SHEP-80)

## Problem Statement

Currently, our `sync_boostr_data` script only syncs deals that are in the Closed Won stage (100%). However, the Ad Ops team also includes deals in the Renewal and Verbal stages in their calculations, so we need to start syncing these as well.

## Proposed Changes

- Adds a new column, Stage Name, to the `BoostrDeal` schema to record the current confidence stage of a deal's likelihood of success
- Start syncing deals that have a high percentage of succeeding (> 75%):
   - 75% Reneway
   - 90% Verbal
   - Closed Won (we were only syncing this one before)

## Verification Steps

- Run the `sync_boostr_script` with the `--full-sync` flag. This will resync all deals and when the script finishes running, you should see all deals have a stage name (Verbal / Closed Won / Renewal).

Note: Once this PR is merged, I will run the script on prod and stage to backfill the stage names for all the previous deals.


[SHEP-80]: https://mozilla-hub.atlassian.net/browse/SHEP-80?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
